packages/multi-test/src/app.rs

```diff
@@ -19,153 +19,345 @@ use crate::wasm::{ContractData, Wasm, WasmKeeper};
 use crate::BankKeeper;
 
 use anyhow::Result as AnyResult;
-use derivative::Derivative;
 
 pub fn next_block(block: &mut BlockInfo) {
     block.time = block.time.plus_seconds(5);
     block.height += 1;
 }
 
+/// Type alias for default build `App` to make its storing simpler in typical scenario
+pub type BasicApp<ExecC = Empty, QueryC = Empty> = App<
+    BankKeeper,
+    MockApi,
```

Comment:
I would say MockApi and MockStorage could remain dyn if this helps simplify.

However, in practice is seems Wasm and custom types are the hardest part to deal with.

Reply:
`MockApi` and `MockStorage` took me maybe 2h in total. It was straightforward substitution in all places, no wired dependencies.